### PR TITLE
Fix Broken Recipes

### PR DESCRIPTION
**Pull request by Kanzaji**
While doing EE setup, I've found some broken recipes ^^"

Some random non existent ore:
obraz
obraz
This should probably get reported to Create ^^" If can be recreated with just BYG and Create (as ID of this recipe is BYG Compat)

Explorer Compass using wrong tag:
obraz

BYG Broken sandstones recipes:
obraz
This was already reported.. but BYG dev can't reproduce it? Anyhow its not fixed, so I've fixed it manually

PS: I have no idea from where that commit with changelog came >.>

### CODE REVIEW

**Comment by Kanzaji** [comment]
Woops, I forgot to mention pigment recipes ^^"

BYG Compat for those is broken >.< But most of them was already hidden, except two, which I've modified
Forgot to take screenshot of those :x

**Comment by Kanzaji** [comment]
Woops, and this now has conflicts 😅

File: kubejs/server_scripts/base/recipes/enigmatica/remove.js [hunk 1]
```diff
@@ -77,13 +77,12 @@ ServerEvents.recipes((event) => {
...
         { type: 'immersiveengineering:alloy' },
-
         { type: 'thermal:sawmill' },
         { type: 'thermal:pulverizer_recycle' },
         { type: 'thermal:insolator' },
         { type: 'thermal:compression_fuel' },
-
-        { mod: 'gateways' }
+        { mod: 'gateways' },
+        { id: "create:compat/byg/crushing/lignite_ore"},
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/base/recipes/enigmatica/remove.js:85]: This should be in the section that holds id removals in this array

File: kubejs/server_scripts/base/recipes/mekanism/pigmentextractor.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+ServerEvents.recipes((event) => {
```
Comment (NielsPilgaard) [inline kubejs/server_scripts/base/recipes/mekanism/pigmentextractor.js:1]: File name should match recipe type `pigment_extracting`

**Review by NielsPilgaard — CHANGES_REQUESTED** [review]

**Review by NielsPilgaard — APPROVED** [review]

**Pull request merged**